CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ Changelog
 in development
 --------------
 
+Fixed
+~~~~~
+
+* Fix Type error for ``time_diff`` critera comparison. convert the timediff value as float to match
+  ``timedelta.total_seconds()`` return.
```

Comment:
Minor comment - if you look at the other items in the change log, can you add #5462 to the end of the description of the change request. That way we can match changes to the PRs that introduced them, helps us find out changes made at a later point.

Reply:
Ok, i'll make the update. and write UT too.